pkg/pillar/cmd/vaultmgr/vaultmgr.go

```diff
@@ -182,11 +182,13 @@ func getProtectorID(vaultPath string) ([][]string, error) {
 func changeProtector(vaultPath string) error {
 	protectorID, err := getProtectorID(vaultPath)
 	if protectorID != nil {
-		if err := stageKey(true, oldKeyDir, oldKeyFile); err != nil {
+		//cloudKeyOnlyMode=true, useSealedKey=false
+		if err := stageKey(true, false, oldKeyDir, oldKeyFile); err != nil {
 			return err
 		}
 		defer unstageKey(oldKeyDir, oldKeyFile)
```

Comment:
What does this do? Please add some comments.

Do we still need the cloudKeyOnly going forward?

Reply:
changeProtector is used on deprecated vaults. It is used for migrating them to TPM based keys, from cloudOnlyKey (which was a bug introduced in the late 2019). We still need to keep cloudKeyOnly mode for some more time, till we migrate all the deprecated vaults to TPM based keys, depending on how old is the image from which we are getting upgraded.

Reply:
Added a comment to explain the usage of this method